### PR TITLE
feat: add basic page templates

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Adding the following new templates:

- Basic page template (EN)
- Basic page template (FR)
- Basic page table of contents template (EN)
- Basic page table of contents template (FR)

[Zenhub ticket](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/1047)

Note: Holding off from merging this PR until design is ready to release.

### CODE REVIEW

File: templates/english/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:1]: Could we change all instances of `TO DO` to `TODO`? I believe the standard usage is the first one, and I know IDEs also grab `TODO` and create a nice list out of it
```suggestion
<!-- TODO: Remove all comments before deploying your code to production. -->
```

**Review by daine — CHANGES_REQUESTED** [review]
While we're waiting for design, just have a small text change request for `TODO`

File: templates/english/basic-page-template-table-of-contents.html
Comment (melaniebmn) [reply templates/english/basic-page-template-table-of-contents.html]: All updated @daine :)

**Review by daine — APPROVED** [review]
LGTM! 

Don't merge; will coordinate with design once the figma design is ready [Zenhub ticket](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/1132)

File: templates/english/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/english/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/english/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/english/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

File: templates/english/basic-page-template.html
Comment (daine) [inline templates/english/basic-page-template.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/english/basic-page-template.html
Comment (daine) [inline templates/english/basic-page-template.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/english/basic-page-template.html
Comment (daine) [inline templates/english/basic-page-template.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/english/basic-page-template.html
Comment (daine) [inline templates/english/basic-page-template.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

File: templates/french/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/french/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/french/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/french/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

File: templates/french/basic-page-template.html
Comment (daine) [inline templates/french/basic-page-template.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/french/basic-page-template.html
Comment (daine) [inline templates/french/basic-page-template.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/french/basic-page-template.html
Comment (daine) [inline templates/french/basic-page-template.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/french/basic-page-template.html
Comment (daine) [inline templates/french/basic-page-template.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

**Review by daine — CHANGES_REQUESTED** [review]
We need to update the versions to the newest before we merge this in

File: templates/english/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:18]:
```suggestion
    <!-- TODO: Replace <version-number> with the latest version number to receive future updates of the GC Design System utility framework. All notable changes will be documented in the changelog: https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md -->
```

File: templates/english/basic-page-template-table-of-contents.html
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@<version-number>/dist/gcds-utility.min.css"
```

**Review by daine — CHANGES_REQUESTED** [review]
Small request to change nubmer to number 😅

File: templates/english/basic-page-template-table-of-contents.html
Comment (melaniebmn) [reply templates/english/basic-page-template-table-of-contents.html]: Argh oops - should be fixed!

File: templates/english/basic-page-template-table-of-contents.html
Comment (melaniebmn) [reply templates/english/basic-page-template-table-of-contents.html]: Should be fixed now!

**Review by daine — DISMISSED** [review]

**Review by daine — DISMISSED** [review]
LGTM

**Comment by melaniebmn** [comment]
@daine updated the basic page templates with the new spacing classes so I will need (hopefully) one last approval for this PR.  :crossed_fingers:

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Review by daine — APPROVED** [review]

**Pull request merged**
